src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -1144,6 +1311,37 @@ begin
   simp only [add_assoc, ennreal.add_halves, le_refl],
 end
 
+/-- One implication of the portmanteau theorem:
+Weak convergence of probability measures implies that the limsup of the measures of any closed
+set is at most the measure of the closed set under the limit probability measure.
+-/
+lemma probability_measure.limsup_measure_closed_le_of_tendsto
+  {α ι : Type*} {L : filter ι}
+  [measurable_space α] [pseudo_emetric_space α] [opens_measurable_space α]
+  {μ : probability_measure α} {μs : ι → probability_measure α}
+  (μs_lim : tendsto μs L (𝓝 μ)) {F : set α} (F_closed : is_closed F) :
+  L.limsup (λ i, (μs i : measure α) F) ≤ (μ : measure α) F :=
+by apply finite_measure.limsup_measure_closed_le_of_tendsto
+         ((probability_measure.tendsto_nhds_iff_to_finite_measures_tendsto_nhds L).mp μs_lim)
+         F_closed
+
+/-- One implication of the portmanteau theorem:
+Weak convergence of probability measures implies that the liminf of the measures of any open set
+is at least the measure of the open set under the limit probability measure.
+-/
+lemma probability_measure.le_liminf_measure_open_of_tendsto
+  {α ι : Type*} {L : filter ι}
+  [measurable_space α] [pseudo_emetric_space α] [opens_measurable_space α]
+  {μ : probability_measure α} {μs : ι → probability_measure α}
+  (μs_lim : tendsto μs L (𝓝 μ)) {G : set α} (G_open : is_open G) :
+  (μ : measure α) G ≤ L.liminf (λ i, (μs i : measure α) G) :=
+begin
+  have h_closeds : ∀ F, is_closed F → L.limsup (λ i, (μs i : measure α) F) ≤ (μ : measure α) F,
+    from λ F F_closed, probability_measure.limsup_measure_closed_le_of_tendsto μs_lim F_closed,
+  exact is_probability_measure.le_measure_liminf_of_limsup_measure_compl_le
+        G_open.measurable_set (h_closeds _ (is_closed_compl_iff.mpr G_open)),
+end
+
```

Comment:
could you also add here the version corresponding to tendsto_measure_of_null_frontier, for sets whose boundary has zero measure? (Or is it planned in another PR, maybe?)

Reply:
I'll add it, it seems natural in this PR now that other null frontier things were included as well.